classes/dpkg-bazel.bbclass

```diff
@@ -27,9 +27,16 @@ XZ_DEP = "xz-utils"
 
 # fetch all bazel deps and add to package sources
 bazel_fetch() {
+    deb_dl_dir_import "${WORKDIR}/rootfs" ${HOST_BASE_DISTRO}-${BASE_DISTRO_CODENAME}
+
+    deb_dir="/var/cache/apt/archives"
+    ext_root="${PP}/rootfs"
+    ext_deb_dir="${ext_root}${deb_dir}"
+
     E="${@ isar_export_proxies(d)}"
     schroot -d / -c ${SBUILD_CHROOT} -u root -- sh <<EOF
         set -e
+        cp -n --no-preserve=owner ${ext_deb_dir}/*.deb -t ${deb_dir}/ || :
```

Comment:
can we please use hard-links here. These packages are not small (esp. the dbgsym variants) and locally I already ran out of space due to all the copies we make during the build.

Reply:
Discuss this with Isar. This here is just a copy of what upstream does.

Reply:
Ok. I'll do. Then I'll approve this and you can merge it.